src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,14 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
```suggestion
  const { name, email } = user;
```
You can write it in one line as there are only 2 props and the line is quite short